src/Resources/config/routing/api_nelmio_v3.xml

```diff
@@ -1,39 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing https://symfony.com/schema/routing/routing-1.0.xsd">
-    <route id="sonata_api_user_group_delete_group" path="/group/{id}.{_format}" methods="DELETE" controller="Sonata\UserBundle\Controller\Api\GroupController::deleteGroupAction" format="json">
+    <import type="xml" resource="./api_nelmio3/group.xml" name-prefix="sonata_api_user_group_"/>
+    <import type="xml" resource="./api_nelmio3/user.xml" name-prefix="sonata_api_user_user_"/>
+    <!-- Wrong routes introduced in version 4.6.0, to be removed in next major. -->
```

Comment:
```suggestion
    <!-- NEXT_MAJOR: Remove these wrong routes introduced in version 4.6.0 -->
```

`NEXT_MAJOR` is really helpful to find all the TODO for a next major.
Before releasing a new major, we just have to look for `NEXT_MAJOR` to know if it's ready or not.